documentation/content/el/books/handbook/desktop/_index.adoc

```diff
@@ -724,7 +724,7 @@ endif::[]
 
 [source,shell]
 ....
-# pkg_add -r kmymoney2
+# pkg install kmymoney2
```

Comment:
This port died around two decades ago, 

<https://www.freshports.org/deskutils/kmymoney2/>

Reply:
maybe gnucash could be an acceptable alternative?

Reply:
actually, if we drop the 2, we're right there where we need to be: https://www.freshports.org/finance/kmymoney/

Reply:
_gnucash_ already at https://github.com/freebsd/freebsd-doc/pull/284/files#diff-55ac5eb437c8f4de5f81117cf740a03b14b3649b79e79907ad22687d5494f8a4L670-R657